content/spin/v2/kv-store-api-guide.md

```diff
@@ -85,38 +85,32 @@ fn handle_request(_req: Request) -> Result<impl IntoResponse> {
 
 > [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/)
 
-With Typescript, the key value functions can be accessed after opening a store using either [the `Kv.open` or the `Kv.openDefault` methods](https://fermyon.github.io/spin-js-sdk/variables/Kv.html) which returns a [handle to the store](https://fermyon.github.io/spin-js-sdk/interfaces/_internal_.KvStore.html). For example:
+With Typescript, the key value functions can be accessed after opening a store using either [the `Kv.open` or the `Kv.openDefault` methods](https://fermyon.github.io/spin-js-sdk/modules/Kv.html) which returns a [handle to the store](https://fermyon.github.io/spin-js-sdk/interfaces/Kv.Store.html). For example:
 
-```javascript
-import { HandleRequest, HttpRequest, HttpResponse, Kv } from "@fermyon/spin-sdk"
+```ts
+import { ResponseBuilder } from "@fermyon/spin-sdk";
 
-const encoder = new TextEncoder()
-
-export const handleRequest: HandleRequest = async function (request: HttpRequest): Promise<HttpResponse> {
+export async function handler(req: Request, res: ResponseBuilder) {
     let store = Kv.openDefault()
     store.set("mykey", "myvalue")
-    return {
-            status: 200,
-            headers: {"content-type":"text/plain"},
-            body: store.get("mykey") ?? encoder.encode("Key not found")
-    }
+    res.status(200)
+    res.set({"content-type":"text/plain"})
+    res.send(store.get("mykey") ?? "Key not found")
```

Comment:
Previously this had an `encoder.encode` around the `"Key not found"` because bytes vs strings.  Is that no longer needed?

Reply:
nope, `res.send()` takes both bytes and string.